modules/apiserversource-yaml.adoc

```diff
@@ -64,16 +65,18 @@ subjects:
 If you want to re-use an existing service account with the appropriate permissions, you must modify the `authentication.yaml` for that service account.
 ====
 +
-After you have created the `authentication.yaml` file, apply it by entering the following command:
+After you have created the `authentication.yaml` file, apply it:
 +
+
+[source,terminal]
 ----
-$ oc apply --filename authentication.yaml
+$ oc apply -f authentication.yaml
```

Comment:
Should these files have 3-letter extensions (`.yml`) or does that even matter?

Reply:
I think either one is fine, I'll leave as .yaml for now for consistency since there are a ton of them, I don't want to change just one.